ingestion/functions/parsing/brazil_amapa/amapa.py

```diff
@@ -0,0 +1,167 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if ('lambda' not in sys.argv[0]):
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+import parsing_lib
+
+_DATE_CONFIRMED_INDEX = 0
+_HEALTHCARE_WORKER_INDEX = 1
+_PREEXISTING_CONDITIONS_INDEX = 2
+_METHOD_CONFIRMATION_INDEX = 3
+_CONFIRMATION_INDEX = 4
+_GENDER_INDEX = 5
+_NOTES_BAIRRO_INDEX = 6
+_MUNICIPALITY_INDEX = 7
+_AGE_INDEX = 8
+_ETHNICITY_INDEX = 9
+_NOTES_INDIGENOUS_GROUP_INDEX = 10
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "Masculino":
+        return "Male"
+    elif raw_gender == "Feminino":
+        return "Female"
+    # TODO(AB, Anya): Handle ensuring None fields aren't sent in requests.
+    return "Other"
+
+def convert_age(age: float):
+    return {
+        "start": age,
+        "end": age
+    }
+
+def convert_confirmation_method(raw_test: str):
+    if raw_test == "RT-PCR":
+        return "PCR test"
+    elif "TESTE" in raw_test:
+        return "Serological test"
+    return "Unknown"
```

Comment:
maybe `print(f'unknown confirmation method: {raw_test}')` before returning so that when we look at the logs we know what we're missing.